dear_petition/petition/models.py

```diff
@@ -402,18 +403,13 @@ class PetitionDocument(PrintableModelMixin, models.Model):
         "self", on_delete=models.CASCADE, null=True, related_name="following_document"
     )
     agencies = models.ManyToManyField(Contact, related_name="+")
+    form_type = models.CharField(choices=FORM_TYPES, max_length=255)
+    form_specific_data = models.JSONField(default=dict)
```

Comment:
I'd like @copelco's input on this, but something about this `form_specific_data` field seems off to me. I think it's because it's so vague that any data could be thrown in there and the keys are arbitrary. In this case, you are using it store `is_checkmark_3b_checked`. But there are multiple 3B checkboxes in this dismissal form. The one you're setting is under the `Petition to Expunge` section and the other one is under `Findings of Fact`. They even have similar language.

I think my overall point is that we'd ideally build out a more robust solution. Maybe subclasses of `PetitionDocument`? Doesn't necessarily need to be done in this PR, but I think it's something we should do after this feature gets merged.